DSharpPlus.CommandsNext/CommandsNextExtension.cs

```diff
@@ -43,7 +43,7 @@
 namespace DSharpPlus.CommandsNext
 {
     /// <summary>
-    /// This is the class which handles command registration, management, and execution. 
+    /// This is the class which handles command registration, management, and execution.
```

Comment:
Whitespace removal is accidental, I'll return it back if it's intended to be there.